tests/Client/SymfonyClientTest.php

```diff
@@ -110,7 +100,7 @@ public function it_can_query_solr(\Closure $query, array $expected): void
      */
     public function it_can_update_solr(): void
     {
-        $client = SolrClient::create(['base_url' => getenv('SOLR_URL')]);
+        $client = SolrClient::create(['base_uri' => getenv('SOLR_URL')]);
```

Comment:
A README-ben a példát is átírnád légyszi, elfelejtettem.

Reply:
thx!